Formula/percolator.rb

```diff
@@ -3,39 +3,22 @@ class Percolator < Formula
   desc "Semi-supervised learning for peptide identification from shotgun proteomics data"
   homepage "http://percolator.ms"
   url "https://github.com/percolator/percolator/archive/rel-3-05.tar.gz"
+  version "3.05"
   sha256 "5b746bdc0119a40f96bc090e02e27670f91eeb341736911750b170da7e5c06bb"
   license all_of: ["Apache-2.0", "BSD-2-Clause", "BSD-3-Clause", "MIT"]
 
   depends_on "boost" => :build
   depends_on "cmake" => :build
-  depends_on "gcc" => :build
-  depends_on "xerces-c" => :build
```

Comment:
Since I'm 99% sure percolator depends on `xerces-c`, how does bottling work without listing it as a dependency?

Reply:
Nvmd -- I actually realized what happened. When you cleaned up the cmake call, `-DXML_SUPPORT=ON` got lost, which disables the XML support (and hence the dependencies on `xerces-c`, `xsd`, etc.)

Reply:
Feel free to push a commit that adds support for XML back in.

Reply:
It's always nice to start from the solid foundation of a working 🍏 PR.